wfl/generate_configs/minim.py

```diff
@@ -201,16 +189,41 @@ def process_step():
                 print('final symmetry group number {}, international (Hermann-Mauguin) {} Hall {} prec {}'.format(
                     dataset['number'], dataset['international'], dataset['hall'], 0.01))
 
-        # Note that if resampling doesn't include original last config, later
-        # steps won't be able to identify those configs as the (perhaps unconverged) minima.
-        # Perhaps status should be set after resampling?
-        traj = _resample_traj(traj, traj_equispaced_n)
 
         if update_config_type:
             # save config_type
             for at0 in traj:
                 config_type_append(at0, at0.info['minim_config_type'])
 
-        all_trajs.append(traj)
+        if traj_subselect is not None:
```

Comment:
Probably simpler to have `_resample_traj` deal with `None`, given all the other `if elseif` we'll need for various resampling criteria.

Reply:
changed in the last commit